scripts/kconfig/kconfigfunctions.py

```
@@ -752,7 +752,7 @@ def dt_compat_on_bus(kconf, _, compat, bus):
 def dt_compat_any_has_prop(kconf, _, compat, prop):
     """
     This function takes a 'compat' and a 'prop' and returns "y" if any
-    node with compatible 'compat' also has a valid property 'prop'.
+    "enabled" node with compatible 'compat' also has a valid property 'prop'.
```

why the quoting of `"enabled"` ?

```suggestion
    enabled node with compatible 'compat' also has a valid property 'prop'.
```

---

I guess because it makes it clear that "enabled" is an explicit value, the other functions on this file do this, wanted to be consistent

---

> the other functions on this file do this, wanted to be consistent

A :+1: for trying to be consistent, but I guess the existing code on main is already a bit inconsistent on this :smile: 

https://github.com/zephyrproject-rtos/zephyr/blob/0237d375de5fe8912b9e4134f45f7774a155b977/scripts/kconfig/kconfigfunctions.py#L81
https://github.com/zephyrproject-rtos/zephyr/blob/0237d375de5fe8912b9e4134f45f7774a155b977/scripts/kconfig/kconfigfunctions.py#L122
https://github.com/zephyrproject-rtos/zephyr/blob/0237d375de5fe8912b9e4134f45f7774a155b977/scripts/kconfig/kconfigfunctions.py#L741
https://github.com/zephyrproject-rtos/zephyr/blob/0237d375de5fe8912b9e4134f45f7774a155b977/scripts/kconfig/kconfigfunctions.py#L808
https://github.com/zephyrproject-rtos/zephyr/blob/0237d375de5fe8912b9e4134f45f7774a155b977/scripts/kconfig/kconfigfunctions.py#L877

Both `enabled` and `"enabled"` will be accepted, but if feel free to align whole file to a consistent style :wink: